c-pallets/file-bank/src/lib.rs

```diff
@@ -274,6 +279,7 @@ pub mod pallet {
 
 	#[pallet::storage]
 	#[pallet::getter(fn filler_index_count)]
+	// Some reason why CountedStorageMap wasn't used instead for FillerMap?
```

Comment:
The reason why CounterStorageMap is not used here is that I need to record an index that only grows to ensure the uniqueness of each file index.

Avoid using CounterStorageMap to reduce count and duplicate indexes when a file is deleted